### PR TITLE
maria db backend test

### PR DESCRIPTION
**Pull request by dnErf**
@aElian01  please... thanks...

### CODE REVIEW

File: server/tests/maria.test.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+describe('should be able to connect and communicate with maria', () => {
+  beforeAll(async () => {
+    mdb = await maria.createConnection({
+      host: 'localhost',
+      user: 'root',
+      password: 'mar1a',
+      database: 'hrr'
+    });
+  })
+  test('this should be invoke if before all is successful', () => {
```
Comment (aElian) [inline server/tests/maria.test.js]: minor edit: 'this should be invoked if beforeAll is successful'

File: server/tests/maria.test.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    expect('maria').toBe('maria');
+  });
+  test('should be able to get a playlist base on id', () => {
+    mdb
+      .query('SELECT _id FROM current_playlist WHERE _id = 0')
+      .then((results) => {
+        console.log('results', results);
+        expects(results).toBe(results);
```
Comment (aElian) [inline server/tests/maria.test.js]: expects(results).toBe(results) is checking results === results? I think you want to be checking against an expected value...

File: server/tests/maria.test.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      });
+  });
+  test('should be able to make request on api and get the playlist', () => {
+    axios
+      .get('http://localhost:3002/api/playlist/1')
+      .then((results) => {
+        expects(results).toBe(results);
```
Comment (aElian) [inline server/tests/maria.test.js]: check comment above :)

File: server/tests/maria.test.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      });
+  });
+  test('should be able to delete and insert song on the playlist', () => {
+    axios
+      .get('http://localhost:3002/api/playlist/4')
+      .then(async (results) => {
+        let parsed_results = JSON.parse.results;
```
Comment (aElian) [inline server/tests/maria.test.js]: JSON.parse(results)?

Comment (dnErf) [reply server/tests/maria.test.js]: thanks eliana.. no shortcuts for me then.. lol..

**Comment by dnErf** [comment]
@aElian01  i submitted the fix for the comments above ... i will need to merge this now because i am having an issues with my branches .. thanks for the good review ..

**Pull request merged**